toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -188,12 +188,19 @@ func PrintBuildSummary(pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMutex, bu
 	logger.Log.Infof("Number of failed SRPMs:            %d", len(failures))
 	logger.Log.Infof("Number of blocked SRPMs:           %d", len(unbuiltSRPMs))
 	logger.Log.Infof("Number of unresolved dependencies: %d", len(unresolvedDependencies))
+
+	if allowToolchainRebuilds && (len(rpmConflicts) > 0 || len(srpmConflicts) > 0) {
+		logger.Log.Infof("Toolchain RPMs conflicts are ignored since ALLOW_TOOLCHAIN_REBUILDS=y")
+	}
+
+	conflictsLogger := logger.Log.Errorf
+	if allowToolchainRebuilds || (len(rpmConflicts) == 0 && len(srpmConflicts) == 0) {
+		conflictsLogger = logger.Log.Infof
+	}
```

Comment:
Nit-pick: could we move it a few lines above where we define other variables. It seems that for this function we're splitting the definition and usage, so let's not mix in a different approach now.